content/guides/dam-connection/synchronize-assets.md

```diff
@@ -1,49 +1,49 @@
 
 ![Macro overview of the DAM connection](../../img/guides/dam-connection-macro.svg)
 
 To be crystal clear, the goal of the connector is to collect assets from the DAM and send them to the PIM. Easy peasy, right?  
 
 Well then! Let's zoom in on what's inside that connector.  
-Below is the general diagram of what your connector will actually be doing under the hood.
+Below is the general diagram of what your connector will actually do under the hood.
 
 ![General workflow](../../img/guides/dam_pim-general-workflow.svg)
 
-It can be summed up into 3 steps:
+It can be summed up in 3 steps:
 - First, collect your DAM assets,
 - Then, turn them into PIM assets,
-- To finish, create those PIM assets in your PIM.
+- To finish, create these PIM assets in your PIM.
 
 Tada! :tada:
 
 ## Collect assets from the DAM
 
 The first step of the synchronization is to collect the available assets from the DAM.
 
-Depending on your DAM capabilities, there are multiple ways to do this (API calls, message queue, ...).
+Depending on your DAM capabilities, there are multiple ways to do it (API calls, message queue, ...).
 
 See the [Define your technical stack](technical-stack.html) part of this documentation to have an idea of the different possibilities and their main constraints.
 
-To collect those assets, you'll need to first fetch the codes of the PIM asset families that you defined earlier in the [PIM structuration](pre-requisites.html#in-the-pim) step. To do so, you can either:
+To collect these assets, first you'll need to fetch the codes of the PIM asset families that you defined earlier in the [PIM structuration](pre-requisites.html#in-the-pim) step. To do so, you can either:
 - call the PIM API [asset family GET endpoint](/api-reference-asset-manager.html#get_asset_families),
 - write directly the codes of your asset families in a configuration file, the way we did in our [skeleton](https://github.com/akeneo/dam-connector/blob/master/config/resources/dam-example/mapping.yaml).
 
-Thanks to the asset family codes, you'll now be able to ask the DAM only for the assets that belong to those families. This is thanks to the preparation you operated on your DAM side by [adding the asset family information on each of your assets](pre-requisites.html#which-asset-family-your-dam-product-assets-belong-to). It will avoid requesting your entire DAM, which would be really useless and counterproductive. :wink:
+Thanks to the asset family codes, you'll now be able to only call from the DAM the assets that belong to these families. This is thanks to the settings you operated on the DAM side by [adding the asset family information on each of your assets](pre-requisites.html#which-asset-family-your-dam-product-assets-belong-to). It will avoid requesting the entire DAM data, which would be really useless and counterproductive. :wink:
```

Comment:
```suggestion
Thanks to the asset family codes, now, you'll be able to ask the DAM only for the assets that belong to those families. This is thanks to the upstream work you operated on the DAM side by [adding the asset family information on each of your assets](pre-requisites.html#which-asset-family-your-dam-product-assets-belong-to). It will avoid requesting the entire DAM data, which would be really useless and counterproductive. :wink:
```